accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -151,6 +524,10 @@ Windows-specific:
 
 ### System.IO.Pipes
 
+* **AnonymousPipeServerStreamAcl**
+    - `Create(PipeDirection, HandleInheritability, Int32, PipeSecurity)`
+* **NamedPipeServerStreamAcl**
+    - `Create(String, PipeDirection, Int32, PipeTransmissionMode, PipeOptions, Int32, Int32, PipeSecurity, HandleInheritability, PipeAccessRights)`
 * **NamedPipeClientStream**
     - `Connect(Int32)`
     - `get_NumberOfServerInstances()`
```

Comment:
I think these APIs and the ones below are not cross platform.
https://github.com/dotnet/runtime/blob/50a999dd3597a33b62a7a1a1712c849e764d50e9/src/libraries/System.IO.Pipes/src/System.IO.Pipes.csproj#L17-L23

Reply:
Actually, is just `NamedPipeClientStream.Connect(Int32)` that doesn't throw PNSException nor is annotated with `MinimumOSPlatformAttribute`.

Reply:
Cool, removed `Connect(Int32)`